### DISCUSSION TITLE
Connect to a remote adapter

### DISCUSSION THREAD

**Discussion by salvovitale** [Koenkk/zigbee2mqtt#10126] [General]
Is it also possible to connect to a remote adapter using ssl? I do not see any ssl configuration in the doc.
https://www.zigbee2mqtt.io/advanced/remote-adapter/connect_to_a_remote_adapter.html#_1-install-ser2net

It would be a nice to have especially if zigbee2mqtt runs outside your local network.

**Comment by mausch** [comment]
[ser2net does support SSL](https://man.archlinux.org/man/community/ser2net/ser2net.yaml.5.en#ENCRYPTION), but I'm not sure that the z2m client does. 
A workaround would be running that over SSH or a VPN.
